### PR TITLE
Item.contains returns false when matrix not invertible instead of blowing up

### PR DESCRIPTION
**Pull request by danfstucky**
### Description
Currently, if an irregular shape is present, it can cause the following error with hover/click events:
`TypeError: Cannot read property 'isInside' of null`
It occurs because the shape is not invertible so the `_inverseTransform` fn returns null which causes the error downstream.
paper.js should return false instead of failing.

#### Related issues
Relates to https://github.com/paperjs/paper.js/pull/1156 (I believe this old PR was trying to address the same issue)

### Checklist
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the JSHint rules (`npm run jshint` passes)

### CODE REVIEW

File: src/item/Item.js [hunk 1]
```diff
@@ -1803,8 +1803,12 @@ new function() { // Injection scope for various item event handlers
...
     contains: function(/* point */) {
         // See CompoundPath#_contains() for the reason for !!
-        return !!this._contains(
-                this._matrix._inverseTransform(Point.read(arguments)));
+        if (this._matrix.isInvertible()) {
+            return !!this._contains(
+                    this._matrix._inverseTransform(Point.read(arguments)));
+        } else {
+            return false;
+        }
```
Comment (lehni) [inline src/item/Item.js]: Could you shorten this to:

```js
var matrix = this._matrix;
return (
	matrix.isInvertible() &&
	!!this._contains(this._matrix._inverseTransform(Point.read(arguments)))
);
```

Comment (lehni) [reply src/item/Item.js]: (the first line is so that minification works more efficiently, because local variables can be minified, while properties can't)

Comment (danfstucky) [reply src/item/Item.js]: Made the change. Didn't know that about the minification too. Thanks.

Comment (lehni) [reply src/item/Item.js]: Thanks!
